src/Catalyst.Core.Lib.Tests/UnitTests/Validators/TransactionValidatorTests.cs

```diff
@@ -160,15 +147,31 @@ public FakeContext(ISignature signature, bool verifyResult)
             public int SignatureLength { get; }
             public int SignatureContextMaxLength { get; }
             public IPrivateKey GeneratePrivateKey() { throw new NotImplementedException(); }
-            public IPublicKey GetPublicKeyFromPrivateKey(IPrivateKey privateKey) { throw new NotImplementedException(); }
+
+            public IPublicKey GetPublicKeyFromPrivateKey(IPrivateKey privateKey)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'GetPublicKeyFromPrivateKey'.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4562326)